starcheck/src/lib/Ska/Starcheck/Obsid.pm

```diff
@@ -1264,7 +1264,8 @@ sub check_star_catalog {
                 or (($type =~ /BOT|ACQ/) && ($c->{"GS_ASPQ$i"} > 40)))
             {
                 push @orange_warn,
-                  sprintf "[%2d] Centroid Perturbation Warning.  %s: ASPQ1 = %2d\n",;
+                  sprintf("[%2d] Centroid Perturbation Warning.  %s: ASPQ1 = %2d\n",
+                      $i, $type, $c->{"GS_ASPQ$i"});
```

Comment:
Fine to toss in this unrelated fix but it should be mentioned in the description and tested.

Reply:
Done.